sass/directives/_headers.scss

```diff
@@ -8,6 +8,31 @@
     @include align-items(center);
   }
 
+  &__headline {
+    @include span-columns(12);
+    padding-right: 20px;
+    padding: $header-padding 0;
+    max-width: $header-width-normal;
```

Comment:
this should probably go on the header--large--left mixin. the centered text headers, like small biz, may have wider text